book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
Please add both `Getters` and `Setters` for `@rentals` in the `Book` class.

Reply:
@SajjadAhmad14  there is no need for that if you read this [toturial](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) carefully you will realize all arrays are defined as Setters only